module-1/lab-data_cleaning/your-code/main.ipynb

```diff
@@ -9,10 +9,12 @@
   },
```

Comment:
1. Good to use info function, it gives you the most important information on missing values and data types in your dataset. You are one of the very few people who have used it. Even if your code execution throws the answer to the question posed, please answer it.
2. Step 10: You say that missing values do not provide valuable info, why? If it's the case, why you haven't drop them? The code for this is missing. 
3. Step 11: Use markdown cell type to insert comments in your notebooks instead of code type. You get an error trying to change ViewCount type to integer, ValueError: Cannot convert non-finite values (NA or inf) to integer. Do you understand the error? In that case, you would have needed to "manipulate" missing values before changing data type.